static/app/views/starfish/views/webServiceView/endpointList.tsx

```diff
@@ -33,10 +33,13 @@ import {DataTitles} from 'sentry/views/starfish/views/spans/types';
 import {EndpointDataRow} from 'sentry/views/starfish/views/webServiceView/endpointDetails';
 
 const COLUMN_TITLES = [
-  'Endpoint',
+  t('Endpoint'),
   DataTitles.throughput,
+  t('Change'),
   DataTitles.p95,
+  t('Change'),
   DataTitles.errorCount,
+  t('Change'),
```

Comment:
Maybe we should slap `Change` into Datatitles too, seems to be used a bunch

Reply:
yup! good idea!